components/31-molecules/accordion/_accordion.scss

```diff
@@ -40,22 +40,25 @@ dl.accordion {
     border-bottom: 2px solid;
 
     h3 {
-      margin: .75rem 0;
+      margin: 1rem 0;
     }
   }
 
   .accordion--button {
     width: 100%;
     padding: 0;
     overflow: visible;
+    display: flex;
```

Comment:
Ik denk dat display altijd bovenaan staat bij de properties. Geeft de sass compiler hier geen opmerkingen over in de terminal?

Reply:
Ik heb de variabelen correct gezet, nu krijg ik geen melding meer in de terminal.